contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -67,7 +67,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 gasOverheadBeforeCallback,
     uint32 gasOverheadAfterCallback,
     int256 fallbackNativePerUnitLink,
-    uint96 fee,
+    uint96 donFee,
```

Comment:
Is this a duplicate use of donFee?  the total fee = donFee + adminFee, right?  I just want to make sure we keep consistent language in all contracts.

Reply:
I don't follow, where is the first of the duplicate?

The Coordinator has donFee, the Router has adminFee